src/Confluent.Kafka/AdminClient.cs

```diff
@@ -363,6 +363,63 @@ private ListConsumerGroupsReport extractListConsumerGroupsResults(IntPtr resultP
             return result;
         }
 
+        private DescribeUserScramCredentialsResult extractDescribeUserScramCredentialsResult(IntPtr eventPtr){
+            var result = new DescribeUserScramCredentialsResult();
+            if (Librdkafka.DescribeUserScramCredentials_result_get_errorcode(eventPtr)!= ErrorCode.NoError)
+            {
+                result.Error = new Error(Librdkafka.DescribeUserScramCredentials_result_get_errorcode(eventPtr));
+            }
+            else
+            {
+                result.Error = new Error(ErrorCode.NoError);
+                var Descriptions = new List<UserScramCredentialsDescription>();
+                int count = Librdkafka.DescribeUserScramCredentials_result_get_count(eventPtr);
+                for(int i=0;i<count;i++)
+                {
+                    var Description = new UserScramCredentialsDescription();
+                    IntPtr c_Description = Librdkafka.DescribeUserScramCredentials_result_get_description(eventPtr,i);
+                    Description.User = PtrToStringUTF8(Librdkafka.UserScramCredentialsDescription_get_user(c_Description));
+                    IntPtr c_error = Librdkafka.UserScramCredentialsDescription_get_error(c_Description);
+
+                    Description.Error = new Error(Librdkafka.error_code(c_error),Librdkafka.error_string(c_error));
+                    if (Librdkafka.error_code(c_error)==0)
+                    {
+                        int num_credentials = Librdkafka.UserScramCredentialsDescription_get_scramcredentialinfo_cnt(c_Description);
+                        var ScramCredentialInfos = new List<ScramCredentialInfo>();
+                        for(int j=0;j<num_credentials;j++)
+                        {
+                            var ScramCredentialInfo = new ScramCredentialInfo();
+                            IntPtr c_ScramCredentialInfo = Librdkafka.UserScramCredentialsDescription_get_scramcredentialinfo(c_Description,j);
+                            ScramCredentialInfo.Mechanism = (ScramMechanism)Librdkafka.ScramCredentialInfo_get_mechanism(c_ScramCredentialInfo);
+                            ScramCredentialInfo.Iterations = Librdkafka.ScramCredentialInfo_get_iterations(c_ScramCredentialInfo);
+                            ScramCredentialInfos.Add(ScramCredentialInfo);
+                        }
+                        Description.ScramCredentialInfos = ScramCredentialInfos;
+                    }
+                    Descriptions.Add(Description);
+                }
```

Comment:
Can we rewrite it using enumerable to keep the code in the same style like being followed in other methods? Something like
```suggestion
            int count = Librdkafka.DescribeUserScramCredentials_result_get_count(eventPtr);
            List<UserScramCredentialsDescription> descriptions = Enumerable.Range(0, count).Select(i =>
            {
                IntPtr c_Description = Librdkafka.DescribeUserScramCredentials_result_get_description(eventPtr, i);
                IntPtr c_error = Librdkafka.UserScramCredentialsDescription_get_error(c_Description);
                ErrorCode errorCode = Librdkafka.error_code(c_error);
                string errorString = Librdkafka.error_string(c_error);

                var description = new UserScramCredentialsDescription
                {
                    User = PtrToStringUTF8(Librdkafka.UserScramCredentialsDescription_get_user(c_Description)),
                    Error = new Error(errorCode, errorString)
                };

                if (errorCode == 0)
                {
                    int numCredentials = Librdkafka.UserScramCredentialsDescription_get_scramcredentialinfo_cnt(c_Description);
                    description.ScramCredentialInfos = Enumerable.Range(0, numCredentials).Select(j =>
                    {
                        IntPtr c_ScramCredentialInfo = Librdkafka.UserScramCredentialsDescription_get_scramcredentialinfo(c_Description, j);
                        return new ScramCredentialInfo
                        {
                            Mechanism = (ScramMechanism)Librdkafka.ScramCredentialInfo_get_mechanism(c_ScramCredentialInfo),
                            Iterations = Librdkafka.ScramCredentialInfo_get_iterations(c_ScramCredentialInfo)
                        };
                    }).ToList();
                }

                return description;
            }).ToList();

            return new DescribeUserScramCredentialsResult
            {
                Error = new Error(ErrorCode.NoError),
                UserScramCredentialsDescriptions = descriptions
            };

```